internal/venice-common/src/main/java/com/linkedin/venice/blobtransfer/P2PBlobTransferManager.java

```diff
@@ -21,4 +23,8 @@ public interface P2PBlobTransferManager<T> extends BlobTransferManager<T> {
   default CompletionStage<T> put(String storeName, int version, int partition) {
     throw new VeniceException("PUT method is not supported for P2P Blob Transfer");
   }
+
+  CompletionStage<? extends InputStream> get(String storeName, int version, int partition)
```

Comment:
this is useless. I don't know why I have it at the first place. Will remove it in next reivision